### DISCUSSION TITLE
ToDos

### DISCUSSION THREAD

**Discussion by cloudcompute** [marbemac/ssrx#35] [Q&A]
Hi @marbemac 

There are few ToDos in this code. Do you have any plams to fix them.

By the way, implementation looks neat.

**Comment by marbemac** [comment]
Hi @cloudcompute, thanks! Re the todos - the ones in the examples are more like notes (perhaps @todo was not best choice), and I don't plan to adjust them unless the underlying issue is fixed in the 3rd party lib or there is some other reason to.

The 2 todos in the `@ssrx/vite` package are legit but I don't have concrete plans to make those improvements at this time - I will consider opportunistically resolving them whenever I dig in to do work on that package though. I could break them out into feature issues in GH I suppose 🤔.

**Reply by cloudcompute** [reply]
Thanks for the response.
